addons/godot-firebase/firebase/firebase.gd

```diff
@@ -52,3 +54,6 @@ func _ready() -> void:
     Auth.connect("login_succeeded", Storage, "_on_FirebaseAuth_login_succeeded")
     Auth.connect("signup_succeeded", Storage, "_on_FirebaseAuth_login_succeeded")
     Auth.connect("token_refresh_succeeded", Storage, "_on_FirebaseAuth_token_refresh_succeeded")
+    Auth.connect("login_succeeded", DynamicLinks, "_on_FirebaseAuth_login_succeeded")
+    Auth.connect("signup_succeeded", DynamicLinks, "_on_FirebaseAuth_login_succeeded")
+    Auth.connect("token_refresh_succeeded", DynamicLinks, "_on_FirebaseAuth_token_refresh_succeeded")
```

Comment:
Do we want to turn this into a loop and just have it loop over all the children? I'm torn.

Reply:
> Do we want to turn this into a loop and just have it loop over all the children? I'm torn.

I feel the same way, but at some point we might have to since it is starting to get larger. It would make it easier in the long run where we just add a new feature, but we would have to make sure each one has all the correct functions